### DISCUSSION TITLE
Add support for S3 or S3 fuse filesystems

### DISCUSSION THREAD

**Discussion by qxmips** [DirectoryLister/DirectoryLister#544] [Ideas & Features]
Hi! could you consider adding support for S3 directories mounted with https://github.com/s3fs-fuse/s3fs-fuse?
it has an issue reading file/directory size and getMTime(): 
the ability to list S3 buckets would be widely apprecieated!

**Comment by PHLAK** [comment]
We will not be adding support for S3 directly to Directory Lister. However, if you can provide more information on the issue you're seeing we may be able to fix the issue with fuse mounted S3 folders. What errors are you seeing?
